packages/components/rollup.config.js

```diff
@@ -1,4 +1,13 @@
+import url from '@rollup/plugin-url';
```

Comment:
https://favro.com/organization/aee06c3fe6503c804f507dd9/22c630cc02b09f2d6f98b8b3?card=Goo-33024

Created a favro ticket because I would need to change the map package as well -> possible merge conflicts with other PR